examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

```diff
@@ -97,7 +97,11 @@
     "from dask_saturn import SaturnCluster\n",
```

Comment:
<p>thank you for making these variables start with <code>EXAMPLE_</code>  so they're scoped to this. preciate you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>